EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1699,59 +1699,59 @@ QUEST_LV_0300_20160310_001698	Move to Veyrhas Hall with Villager Duncan
 QUEST_LV_0300_20160310_001699	Duncan told you that he collected all the books that he will burn at Veyrhas Hall. It doesn't feel good, but anyways, move to Veyrhas Hall with him.
 QUEST_LV_0300_20160310_001700	You have successfully saved Villager Duncan from the demons. Talk to Villager Duncan.
 QUEST_LV_0300_20160310_001701	Talk to Villager James
-QUEST_LV_0300_20160310_001702	Collecting Deoman's Red Blood
-QUEST_LV_0300_20160310_001703	Villager James says that in order to purify the corrupted power of the goddess, demon's red blood is needed. Defeat the nearby demons to collect their red blood. 
-QUEST_LV_0300_20160310_001704	Deliever it to Villager James
-QUEST_LV_0300_20160310_001705	Obtained demon's red blood, Villager James has asked for. Return to him.
+QUEST_LV_0300_20160310_001702	Collect Demon's Red Blood
+QUEST_LV_0300_20160310_001703	Villager James says that in order to purify the corrupted power of the goddess, the demon's red blood is needed. Defeat the nearby demons to collect their red blood. 
+QUEST_LV_0300_20160310_001704	Deliver it to Villager James
+QUEST_LV_0300_20160310_001705	Obtained demon's red blood requested by Villager James. Return to him.
 QUEST_LV_0300_20160310_001706	Defeat nearby demons and collect %s
-QUEST_LV_0300_20160310_001707	Dirty Trick
-QUEST_LV_0300_20160310_001708	Inspect the Annex Laboratory beyon the Black Wall
+QUEST_LV_0300_20160310_001707	Dirty Tricks
+QUEST_LV_0300_20160310_001708	Inspect the Annex Laboratory beyond the Black Wall
 QUEST_LV_0300_20160310_001709	There is something hidden beyond the black wall. Pass through the black wall to inspect the Annex Laboratory.
 QUEST_LV_0300_20160310_001710	Dominikas is right behind you. And you didn't see him coming. Not sure how long he has been there. Talk to him.
 QUEST_LV_0300_20160310_001711	On the way out from the Annex Laboratory, you came across Priest Dominikas. Talk to him.
 QUEST_LV_0300_20160310_001712	Move to the Principal Corridor
 QUEST_LV_0300_20160310_001713	Priest Dominikas praises you for those who can walk through the black wall is those whom the Divine Tree allowed. He says an important ritual is about to commence and he wants to meet him at the Principal Corridor.
```

Comment:
"Priest Dominikas praises you, for those who can walk through the black wall are those whom the Divine Tree allowed.  He says an important ritual is about to commence, and he wants you to meet him"